src/assets/styles/pikaday_custom.scss

```diff
@@ -0,0 +1,82 @@
+@import "node_modules/pikaday/scss/pikaday.scss";
+
+$pd-day-hover-color: lighten($pd-day-color, 8%);
+
+.calendar-container {
```

Comment:
Two teeny calendar styling comments: for the colors of selected days I think the design is using `$aclu-blue`, and the text inputs have left-aligned text

![image](https://cloud.githubusercontent.com/assets/1724544/24768525/afaede6c-1ab7-11e7-996e-a2082edf981e.png)